scss/6-components/_pagination.scss

```diff
@@ -5,35 +5,40 @@
   color: $cads-language__secondary-text-colour;
 }
 
-.cads-paging {
-  button {
-    color: $cads-language__link-colour;
-    border-color: $cads-language__border-colour;
-    box-shadow: none;
-    font-weight: $cads-font-weight__regular;
-    background-color: transparent;
-    min-width: $cads-interactive-target-size;
-    margin-bottom: 0;
+.cads-paging__button {
+  @include cads-button-defaults();
```

Comment:
New button styles conflicted with pagination styles as they were inheriting from that. Rather than unsetting lots of things I opted to use a mixin to share _just_ the common details for these sorts of buttons. It also fixes a bug as a result as pagination now has a proper focus style.